templates/s3/s3.py

```diff
@@ -757,7 +741,22 @@ def do():
         except:
             self.state.set('status', 'running', 'error')
 
-    def _deploy_minio(self, nodes):
+    def _deploy_minio(self):
```

Comment:
```suggestion
    def _deploy_minio(self, nodes=None):
```

Reply:
So you can still pass the list of node, cause listing all the nodes from the directory is quite an heavy operation